neuralmonkey/encoders/raw_rnn_encoder.py

```diff
@@ -11,9 +12,31 @@
 from neuralmonkey.nn.utils import dropout
 from neuralmonkey.dataset import Dataset
 
-# pylint: disable=invalid-name
-RNNCellTuple = Tuple[tf.contrib.rnn.RNNCell, tf.contrib.rnn.RNNCell]
-# pylint: enable=invalid-name
+
+# pylint: disable=too-few-public-methods
+class RNNSpec(object):
+    def __init__(self,
+                 size: int,
+                 direction: str = 'both',
```

Comment:
Moc nevím, jak v Pythonu funguje enum, ale jak se to bude zadávat v konfiguráku? Není to zbytečná komplikace?

Navíc všude jinde se v téhle situaci používají stringy, tak bych to udělal konzistentně.